src/nuget.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You don't need this file in this scenario. There are no bede packages in the Dapperer project so it would work without supplying a specific source (i.e. the bede one in this scenario

Reply:
removed